nni/retiarii/graph.py

```diff
@@ -539,7 +551,9 @@ def _dump(self) -> Any:
         if self.label is not None:
             ret['label'] = self.label
         return ret
-
+    
+    def __hash__(self) -> int:
+        return hash(self.__repr__())
```

Comment:
better to use: `return hash(id(self))`

Reply:
I found it is duplicated with your implementation. Deleted.